### PR TITLE
Update webbluetooth status

### PR DESCRIPTION
**Pull request by shuangMoz**

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
:+1:

File: features/web-bluetooth.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Web Bluetooth API
 category: apps
 bugzilla: 1204396
-firefox_status: in-development
+firefox_status: Considering
```
Comment (marco-c) [inline features/web-bluetooth.md]: under-consideration

**Comment by marco-c** [comment]
Can you tell why we should change it? Feature PRs without any reference are hard to review.

**Comment by shuangMoz** [comment]
I was told that WebBluetooth API now is low priority. And ekr suggests to change the status to "under-consideration".

**Comment by marco-c** [comment]
Thank you!

**Pull request merged**